### PR TITLE
feat: clean up

### PR DESCRIPTION
**Pull request by mxkaske**
part 1

### CODE REVIEW

File: apps/web/src/components/layout/app-footer.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          <Link
+            href="/legal/terms"
+            className="text-foreground underline underline-offset-4 hover:no-underline"
+          >
+            Terms
+          </Link>
+          <span className="text-muted-foreground/70 mx-1">&bull;</span>
+          <Link
+            href="/legal/privacy"
+            className="text-foreground underline underline-offset-4 hover:no-underline"
```
Comment (thibaultleouay) [inline apps/web/src/components/layout/app-footer.tsx:22]: I think legacy and tos don't need to be on every page when you are logged in

File: apps/web/src/components/marketing/plans.tsx [hunk 2]
```diff
@@ -1,96 +1,44 @@
...
   );
 }
 
-interface Props extends Plan {
+interface Props extends PlanProps {
   className?: string;
 }
 
-function Plan({
+export function Plan({
```
Comment (thibaultleouay) [inline apps/web/src/components/marketing/plans.tsx:33]: We are using it also in setting maybe extract it to components shared plan

**Review by thibaultleouay — APPROVED** [review]

File: apps/web/src/components/layout/app-footer.tsx (hunk 1 above)
Comment (mxkaske) [reply apps/web/src/components/layout/app-footer.tsx:22]: Good point, a lot of dashboard don't even have a footer. Maybe we can move it only to marketing pages

Comment (thibaultleouay) [reply apps/web/src/components/layout/app-footer.tsx:22]: Yes 🔥

File: apps/web/src/components/marketing/plans.tsx (hunk 2 above)
Comment (mxkaske) [reply apps/web/src/components/marketing/plans.tsx:33]: We now use only one component for both, marketing and settings. Will extract it later!

**Pull request merged**